output_template/python/lmnet/utils/demo.py

```diff
@@ -49,19 +49,18 @@ def add_fps(orig, fps):
     return orig
 
 
-def check_range(upper, lower, checked_val):
-    if upper < checked_val:
-        checked_val = upper
-    elif lower > checked_val:
-        checked_val = lower
-    return checked_val
+def clamp(lower, upper, target):
+    if target < lower:
```

Comment:
How about

```
tmp = max(lower, target)
tmp = min(max, target)
return tmp
```

Reply:
What is the advantage of this?

Reply:
less lines and little bit readable.
Umm, but this is my personal opinion.
Not necessary change.

Reply:
How about `return min(upper, max(lower, target))` ?

Reply:
@tfujiwar How do you think about this?

Reply:
> How about return min(upper, max(lower, target))

I think that's the best way.
In general, reducing conditional branches helps to keep code simpler.